glusterapi/common.py

```diff
@@ -1,14 +1,27 @@
+
 import datetime
 import hashlib
+from uuid import UUID
 
 import jwt
 import requests
 
-from exceptions import GlusterApiError
+from glusterapi.exceptions import GlusterApiError
+
+
+def validate_uuid(brick_id):
+    try:
+        UUID(brick_id, version=4)
```

Comment:
You can change the function definition to validate_uuid(value, version=4) and change the UUID call to UUID(value, version). Not that we change the version often but it is cleaner that way.